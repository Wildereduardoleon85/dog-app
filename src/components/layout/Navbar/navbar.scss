header {
  nav {
    box-shadow: 0px 1px 5px 0px grey;
    background-color: var(--primary);
    padding: 0 80px;
    display: flex;
    height: 80px;
    position: fixed;
    width: 100%;
    transition: all 0.5s ease;
    z-index: 2;
    transform: translateY(-120%);
    a {
      background: none;
      display: flex;
      align-items: center;
      border: none;
      outline: none;
      font-size: 1.1rem;
      cursor: pointer;
      font-weight: bold;
      height: 100%;
      padding: 0 10px;
      color: white;
      font-weight: normal;
    }
    a:hover {
      background: var(--primary-darken1);
      transition: all 0.3s ease;
    }
    ul {
      display: flex;
      align-items: center;
      .logo {
        width: 40px;
        img {
          width: 100%;
        }
      }
    }
    ul:first-child {
      li:last-child {
        font-family: "Rampart One", cursive;
        font-size: 2rem;
        color: white;
        margin-left: 20px;
      }
    }
    ul:last-child {
      margin-left: auto;
      li {
        height: 100%;
        display: flex;
        align-items: center;
      }
    }
  }
  .navbar {
    transform: translateY(0);
  }
}
