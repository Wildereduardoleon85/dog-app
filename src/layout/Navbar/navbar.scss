.navbar {
  box-shadow: 0px 1px 5px 0px black;
  padding: 0 80px;
  display: flex;
  height: 80px;
  button {
    background: none;
    border: none;
    outline: none;
    font-size: 1.1rem;
    cursor: pointer;
    font-weight: bold;
    height: 100%;
    padding: 0 10px;
  }
  button:hover {
    color: var(--primary);
    background: var(--light-darken1);
    transition: all 0.3s ease;
  }
  ul {
    display: flex;
    align-items: center;
    .logo {
      width: 40px;
      img {
        width: 100%;
      }
    }
  }
  ul:first-child {
    li:last-child {
      font-family: "Rampart One", cursive;
      font-size: 1.5rem;
      color: var(--primary);
      margin-left: 20px;
    }
  }
  ul:last-child {
    margin-left: auto;
    li {
      height: 100%;
      display: flex;
      align-items: center;
    }
  }
}
