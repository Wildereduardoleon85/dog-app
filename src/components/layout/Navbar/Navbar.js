import React from 'react';
import './navbar.scss';

const Navbar = () => {
    return (
        <nav className="navbar">
            <ul>
                <li className="logo">
                    <img src="/img/footprint.svg" alt="footprint" />
                </li>
                <li>
                    Dogs Lovers
                </li>
            </ul>

            <ul>
                <li>
                    <button>Browse All Breeds</button>
                </li>
                <li>
                    <button>Sub.Breeds</button>
                </li>
            </ul>
        </nav>
    ) 
}

export default Navbar

