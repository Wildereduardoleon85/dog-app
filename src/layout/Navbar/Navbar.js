import React from 'react';
import './navbar.scss';

const Navbar = () => {
    return (
        <nav>
            <ul>
                <li className="logo">
                    <img src="/img/footprint.svg" alt="footprint" />
                </li>
                <li>
                    Dogs Lovers
                </li>
            </ul>
        </nav>
    )
}

export default Navbar
