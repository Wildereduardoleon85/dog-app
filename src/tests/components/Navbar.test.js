import React from 'react';
import { shallow } from "enzyme";
import Navbar from '../../components/layout/Navbar/Navbar';


describe('tests in <Navbar/>', () => {
    test('Should show the component correctly', () => {
        const wrapper = shallow( <Navbar/>)
        expect(wrapper).toMatchSnapshot();
    })
})