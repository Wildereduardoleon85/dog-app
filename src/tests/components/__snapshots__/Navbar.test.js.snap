// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests in <Navbar/> Should show the component correctly 1`] = `
<nav
  className="navbar"
  style={
    Object {
      "height": "100px",
    }
  }
>
  <ul>
    <li
      className="logo"
    >
      <img
        alt="footprint"
        src="/img/footprint(white).svg"
      />
    </li>
    <li>
      Dogs Lovers
    </li>
  </ul>
  <ul>
    <li>
      <a
        href="#home"
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="#gallery"
      >
        Image Gallery
      </a>
    </li>
  </ul>
</nav>
`;
